R/cache_filesystem.R

```diff
@@ -23,7 +25,7 @@
 #'
 #' @export
 #' @inheritParams cache_memory
-cache_filesystem <- function(path, algo = "xxhash64") {
+cache_filesystem <- function(path, algo = "xxhash64", compress=FALSE) {
```

Comment:
Thanks! Could you please follow the style used in the package, in particular spaces should be around operators (http://style.tidyverse.org/syntax.html#spacing).